packages/st2mistral/debian/control

```diff
@@ -4,13 +4,13 @@ Priority: optional
 Maintainer: StackStorm Engineering <opsadmin@stackstorm.com>
 Build-Depends: debhelper (>= 9), python, dh-virtualenv (>= 0.8)
 Standards-Version: 3.9.5
-Homepage: http://stackstorm.com/
-#Vcs-Git: git://git@github.com:StackStorm/st2mistral.git
-#Vcs-Browser: https://github.com/StackStorm/st2mistral.git
+Homepage: <insert the upstream URL, if relevant>
+#Vcs-Git: git://anonscm.debian.org/collab-maint/mistral.git
```

Comment:
all these are insignificant issues. to be honest all descriptions, licensing etc should be fixed.